server/routerlicious/packages/services/src/kafkaProducerFactory.ts

```diff
@@ -34,6 +35,8 @@ export function createProducer(
             } else {
                 winston.error("Kafka Producer emitted an error that is not configured to restart the process.");
                 winston.error(inspect(error));
+                Lumberjack.error("Kafka Producer emitted an error that is not configured to restart the process.");
```

Comment:
Lumberjack makes it simpler and we don't need to have another `Lumberjack.error()` below exclusively for the error message. Instead, do:
```typescript
Lumberjack.error("Kafka Producer emitted an error that is not configured to restart the process.", undefined, error);
```